server/controllers/users.js

```diff
@@ -96,3 +96,46 @@ export function getAllUsers(req, res, next) {
       console.log('could not retrieve users', err);
     });
 }
+
+function toCamelCase(word) {
+  const wordArr = word.split('_').join('');
+  const camelized = wordArr.slice(0, 2) + wordArr[2].toUpperCase() + wordArr.slice(3);
+  return camelized;
+}
+
+function addIfExists(paramObj, key, status) {
+  if (key) {
+    const newParamObj = paramObj;
+    newParamObj[toCamelCase(key)] = JSON.parse(status);
+  }
+}
+
+export function manageUserById(req, res, next) {
+  if (req.body.isBanned === 'true') {
+    req.body.isActive = 'false';
+  }
+
+  const params = {
+    id: req.params.id
+  };
+  //  This is necessary because params keys are camel cased and the req.body is coming in as snake
+  Object.keys(req.body).forEach((key) => {
+    addIfExists(params, key, req.body[key]);
+  });
+
+  if (params.isActive && params.isBanned) {
```

Comment:
Since you're already modifying the incoming params in lines 114 and 115, this will never be triggered. In that case, let's just drop this if statement and implicitly deactivate banned users, where banning takes precedence over activating a user. Will also get rid of the funky return statement requirement.

Reply:
👍 , I had an error. Line 114 and 115 wasn't being invoked because req.body is in snake case.